### ISSUE TITLE
Should this page mention pipelines.yml?

### ISSUE THREAD

**Issue by ctindel** [elastic/logstash#8564]
https://www.elastic.co/guide/en/logstash/6.0/config-setting-files.html

Seems like it might need to be mentioned there.

**Comment by dedemorton** [comment]
Yes, it should. There are a couple of other pages (on my list) that should also mention multiple pipelines.

**Comment by dedemorton** [comment]
@karenzone This is actually a pretty simple change. It's been open from awhile :-(.

Might be a good low-hanging-fruit kind of change for you to work on.

**Comment by dedemorton** [comment]
You should add more info about multiple pipelines to these pages, too: 

https://www.elastic.co/guide/en/logstash/master/running-logstash-command-line.html

https://www.elastic.co/guide/en/logstash/master/advanced-pipeline.html - this topic already mentions multiple pipelines, but it might be worth expanding a bit

**Comment by karenzone** [comment]
I noticed the `log4j2.properties` file when I was poking around in the logstash /config directory.  Should I add it to https://www.elastic.co/guide/en/logstash/6.0/config-setting-files.html, too?
